### DISCUSSION TITLE
关于利用滑动切换键盘但是非指定键盘

### DISCUSSION THREAD

**Discussion by ZZZ7664** [imfuxiao/Hamster#539] [Q&A]
作者你好，我把键盘的空格的右滑改为了切换中文九宫键盘（chineseNumeric）
![image](https://github.com/imfuxiao/Hamster/assets/103173090/502ef9ee-1fd4-41f2-8edb-9aea4975b807)

但是切换完却是符号键盘（非分类符号）

**Comment by g6ai** [comment]
好像是叫chineseNineGrid？

**Comment by imfuxiao** [comment]
见[「手册-keyboardtype-选项」](https://ihsiao.com/apps/hamster/docs/CustomKeyboard.html#keyboardtype-%E9%80%89%E9%A1%B9)
